protocol/writing-resource/put-patch-auxiliary-resource.feature

```diff
@@ -0,0 +1,19 @@
+Feature: PUT or PATCH on auxiliary resources
+
+  Background: Set up container and parse link headers
+    * def testContainer = rootTestContainer.createContainer()
+    * def container = testContainer.createContainer()  
+    * def exampleTurtle = karate.readAsString('../fixtures/example.ttl')
+    * def rdfResource = testContainer.createResource('.ttl', exampleTurtle, 'text/turtle');
+
+  Scenario: PUT auxiliary resource to container
+    * def response = clients.alice.sendAuthorized('GET', container.url, null, null)
+    * def links = parseLinkHeaders(response.headers)
```

Comment:
```suggestion
    Given url container.url
    And headers clients.alice.getAuthHeaders('GET', container.url)
    When method GET
    * def links = parseLinkHeaders(responseHeaders)
```
Why not use the normal GET? I know it is longer but I see no need for the special case of `sendAuthorized()`

Reply:
I did that because I don't think of that as a test in its own right, that's just there to get the link. No strong opinions though.

Reply:
That makes sense to me and I have done similar elsewhere (e.g. every time we create a test container or resource). In this case, I wonder if we should provide overrides to the `sendAuthorized` method so that there are simpler variants making the test easier to read. Let's discuss.